chain/chain/src/chain.rs

```diff
@@ -2352,7 +2359,7 @@ impl<'a> ChainUpdate<'a> {
         {
```

Comment:
let's change the function comment to "Check if header is recent or in the store."